src/session/config.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Get's an environment variable. If the env var is not set and a default value is not
+ * provided, then it is assumed it is a mandatory requirement and an error will be
+ * thrown.
+ */
+const getEnvironmentValue = (key: string, defaultValue?: any): string => {
```

Comment:
this file is not needed, see properties.ts

Reply:
Removed